assets/src/edit-story/components/colorPicker/editablePreview.js

```diff
@@ -60,6 +60,7 @@ function EditablePreview({ label, value, width, format, onChange }) {
   );
 
   // Handle ESC keypress to toggle input field.
+  //eslint-disable-next-line react-hooks/exhaustive-deps
   useKeyDownEffect(wrapperRef, { key: 'esc', editable: true }, disableEditing, [
     isEditing,
```

Comment:
Adding `disableEditing` as the only dependency (as enforced by the rule) breaks the test in `assets/src/edit-story/components/colorPicker/test/colorPicker/footerInteraction.js`. The callback is simply not fired.